docs/resources/dashboard.md

```diff
@@ -8090,12 +8166,20 @@ Optional:
 
 - **apm_query** (Block List, Max: 1) The query to use for this widget. (see [below for nested schema](#nestedblock--widget--timeseries_definition--request--apm_query))
 - **display_type** (String) How the marker lines will look. Possible values are one of {`error`, `warning`, `info`, `ok`} combined with one of {`dashed`, `solid`, `bold`}. Example: `error dashed`.
+<<<<<<< HEAD
+- **formula** (Block List) (see [below for nested schema](#nestedblock--widget--timeseries_definition--request--formula))
+=======
+>>>>>>> origin/master
 - **log_query** (Block List, Max: 1) The query to use for this widget. (see [below for nested schema](#nestedblock--widget--timeseries_definition--request--log_query))
 - **metadata** (Block List) Used to define expression aliases. Multiple `metadata` blocks are allowed with the structure below. (see [below for nested schema](#nestedblock--widget--timeseries_definition--request--metadata))
 - **network_query** (Block List, Max: 1) The query to use for this widget. (see [below for nested schema](#nestedblock--widget--timeseries_definition--request--network_query))
 - **on_right_yaxis** (Boolean) Boolean indicating whether the request will use the right or left Y-Axis.
 - **process_query** (Block List, Max: 1) The process query to use in the widget. The structure of this block is described below. (see [below for nested schema](#nestedblock--widget--timeseries_definition--request--process_query))
 - **q** (String) The metric query to use for this widget.
+<<<<<<< HEAD
```

Comment:
Looks like some merge conflicts got leftover (It may be easier to rm the file and let tfplugindocs redefine it from the latest state)